### ISSUE TITLE
Support typechange status

### ISSUE THREAD

**Issue by ObserverOfTime** [gbprod/tree-sitter-gitcommit#71]
`typechange`: file type changed (regular file, symbolic link or submodule)

### Example

```ini
#	typechange: after/ftplugin/svelte.vim
```

#### Current

```clojure
  (generated_comment ; [13, 0] - [13, 39]
    (filepath) ; [13, 2] - [13, 13]
    (annotation)) ; [13, 14] - [13, 39]
```

#### Expected

```clojure
  (generated_comment ; [13, 0] - [13, 39]
    (change ; [13, 2] - [13, 13]
      kind: (typechange)) ; [13, 2] - [13, 13]
    (filepath)) ; [13, 14] - [13, 39]
 ```

**Comment by gbprod** [comment]
Interesting !
I'll will try to do someting :)

**Comment by gbprod** [comment]
Fixed by #72 :)
